specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2021-07-01-preview/galleryImages.json

```diff
@@ -0,0 +1,425 @@
+            "schema": {
+              "$ref": "#/definitions/galleryimagesListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AzureStackHCI/galleryimages": {
+      "get": {
+        "operationId": "galleryimages_ListBySubscription",
+        "description": "List all galleryimages under the subscription",
+        "x-ms-examples": {
+          "ListGalleryImageByResourceGroup": {
+            "$ref": "./examples/ListGalleryImageBySubscription.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Lists all galleryimages under the subscription.",
+            "schema": {
+              "$ref": "#/definitions/galleryimagesListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "galleryimagesProperties": {
+      "type": "object",
+      "x-ms-client-flatten": true,
+      "properties": {
+        "containerName": {
+          "type": "string",
+          "description": "Container Name for storage container"
+        },
+        "imagePath": {
+          "type": "string",
+          "description": "location of the image the gallery image should be created from"
+        },
+        "osType": {
+          "type": "string",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "x-ms-enum": {
+            "name": "OperatingSystemTypes",
+            "modelAsString": false
+          },
+          "description": "operating system type that the gallery image uses. Expected to be linux or windows"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of the gallery image.",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "InProgress"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningStateEnum",
+            "modelAsString": true
+          }
+        },
+        "resourceName": {
+          "type": "string",
+          "description": "name of the object to be used in moc"
+        },
+        "status": {
+          "$ref": "#/definitions/MOCGalleryImageStatus"
```

Comment:
I am curious, why is it "MOCGalleryImageStatus" as opposed to "GalleryImageStatus" ?